cpp/ql/src/semmle/code/cpp/security/CommandExecution.qll

```diff
@@ -85,8 +85,8 @@ class VarargsExecFunctionCall extends FunctionCall {
    * all the other ones start with the command. */
   private int getCommandIdx() {
     if (
-      getTarget().getQualifiedName().matches("\\_spawn%")
-      or getTarget().getQualifiedName().matches("\\_wspawn%"))
+      getTarget().getName().matches("\\_spawn%")
+      or getTarget().getName().matches("\\_wspawn%"))
```

Comment:
Isn't this a semantic change? It now matches `namespace::_spawn` and similar.

Reply:
I think it's fine, for the same reason as https://github.com/Semmle/ql/pull/1303#discussion_r280804696, but please double-check.